openapi/aggregation-openapi.json

```diff
@@ -540,12 +550,14 @@
             }
           }
         },
-        "tags": ["Info"]
+        "tags": [
+          "Info"
+        ]
       }
     },
     "/v5.2/1/tokens": {
       "get": {
-        "operationId": "TokensController_getTokens",
+        "operationId": "getTokens",
```

Comment:
I am against manually editing the swagger files. I think it should either be done from the type generation script or manually handled with wrapper structs.

Reply:
Currently it will be renamed automatically, according to mapping.json in codegen directory